doc/cephfs/troubleshooting.rst

```diff
@@ -128,6 +128,11 @@ things to do:
 
   That prevents any clients from establishing new sessions with the MDS.
 
+* **Dont tweak max_mds** Changing value of ``max_mds`` FS setting variable is
```

Comment:
I would add this as a precautionary note rather than an assertive statement.

Reply:
Please suggest better a wording. `Careful when tweakig max_mds`?

Reply:
See wording above